how-to/join_kubeflow_ecosystem.md

```diff
@@ -0,0 +1,145 @@
+# Application for a Project to Join the Kubeflow Ecosystem
+
+Please see the [proposals/new-project-join-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Public list of adopters.
+
+
+## Background information
+
```

Comment:
Given that there are many items. What about creating some sections such as Governance, Community, Product ecosystem, etc.

Reply:
We could but I think it's just easier to see the full enumeration of the checklist. If it's chunked, to me it would feel like it's longer. WDYT?

Reply:
yes, is more a personal opinion to be honest, I prefer when information is more structured. 
But I'm good if you want to keep it as it is.